docs/set-up/_index.rst

```diff
@@ -70,11 +70,7 @@ Enable Determined to submit jobs to a Slurm cluster.
 
 .. toctree::
    :hidden:
+   :glob:
 
-   Quick Installation <basic>
-   Deploy on Prem <on-prem/overview>
-   Deploy on AWS <aws/overview>
-   Deploy on GCP <gcp/overview>
-   Deploy on Kubernetes <k8s/overview>
-   Deploy on Slurm/PBS <slurm/overview>
-   Cluster Configuration <cluster-configuration>
+   ./* 
+   ./*/_index 
```

Comment:
i decided to keep "setup-cluster" to leave room for other features in the future that may need "setup" instructions. thus "setup-cluster" indicates setup instructions for setting up (and deploying) a cluster.